Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM php:7.4-apache
+LABEL maintainer="matt_henderson@sil.org"
+
+RUN apt-get update -y && \
+    apt-get install -y --no-install-recommends \
+        unzip \
+        zip \
+        cron \
+# Needed to get various scripts
+        curl \
+# Needed for whenavail
+        netcat \
+# Needed to build php extensions
+        libfreetype6-dev \
+        libjpeg62-turbo-dev \
+        libpng-dev \
+        libonig-dev \
+        libxml2-dev \
+        libcurl4-openssl-dev && \
+# Install and enable, see the README on the docker hub for the image
+    docker-php-ext-configure gd --with-freetype=/usr/include --with-jpeg=/usr/include && \
+    docker-php-ext-install -j$(nproc) gd && \
+    docker-php-ext-install pdo pdo_mysql mbstring xml curl && \
+    docker-php-ext-enable gd pdo pdo_mysql mbstring xml curl && \
+    apt-get remove -y \
+        # Note: Don't remove "libfreetype6-dev". The gd extension depends on it.
+        # Note: Don't remove "libjpeg62-turbo-dev". The gd extension depends on it.
+        # Note: Don't remove "libpng-dev". The gd extension depends on it.
+        libonig-dev \
+        libxml2-dev \
+        libcurl4-openssl-dev && \
+    apt-get autoremove -y && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY --from=composer:latest /usr/bin/composer /usr/local/bin/composer
```

Comment:
this an approach I'm using fwiw:  `COPY --from=wait /wait /wait`

Yours might look something like `COPY --from=docker-whenavail:1.0.2 /usr/local/bin/whenavail /usr/local/bin/whenavail`

Of course it needs to be published to Docker Hub but it's nice because it should be faster and would not require `curl`